app/routes/organen/beheer/index.js

```diff
@@ -5,42 +5,86 @@ import { action } from '@ember/object';
 export default class OrganenbeheerIndexRoute extends Route {
   @service store;
 
+  pageSize = 50;
   queryParams = {
-    filter: { refreshModel: true },
-    page: { refreshModel: true },
-    size: { refreshModel: true },
-    sort: { refreshModel: true },
+    active_sort: { refreshModel: true },
+    inactive_sort: { refreshModel: true },
   };
 
   async model(params) {
     const parentModel = this.modelFor('organen');
 
-    const options = this.getOptions(params, parentModel.bestuurseenheid);
-    const bestuursorganen = await this.store.query('bestuursorgaan', options);
+    const active_options = this.getActiveOptions(
+      params,
+      parentModel.bestuurseenheid
+    );
+    const activeOrganenUnfiltered = await this.store.query(
+      'bestuursorgaan',
+      active_options
+    );
+    const activeOrganen = [];
```

Comment:
the model function is getting long. i would extract line 25 till 33 to a function called getActiveDecretaleOrganen or something